### PR TITLE
Create export and worker queues for events

### PR DESCRIPTION
**Pull request by dkryvi**

### CODE REVIEW

File: app/config/dev.json [hunk 1]
```diff
@@ -14,5 +14,8 @@
       "type": "event",
       "index": "event"
     }
+  },
+  "redis": {
+    "checkStalledJobsInterval": 3600000
```
Comment (akosenko) [inline app/config/dev.json]: What is that option?

File: app/utils/redisQueue.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      clearTimeout(queueInitTimeout);
+      queues[key] = queue;
+      resolve(queue);
+    });
+  } else {
+    resolve(queues[key]);
+  }
+});
+
+const createWorkerQueue = (queueName, host = '127.0.0.1', port = 6379) => {
```
Comment (akosenko) [inline app/utils/redisQueue.js]: move host and port to config

File: app/services/exportEventsService.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const logger = require('../utils/logger')('exportEvevntsService');
+const { createExportQueue } = require('../utils/redisQueue');
+
+let eventsQueue;
+
+const initExportEventsQueue = () => createExportQueue('eventsQueue')
```
Comment (akosenko) [inline app/services/exportEventsService.js]: please rename all files and methods from 
*ExportEventsQueue to the *EventsQueue
because export is redunant

**Review by akosenko — CHANGES_REQUESTED** [review]

File: app/utils/redisQueue.js (hunk 2 above)
Comment (dkryvi) [reply app/utils/redisQueue.js]: [Done](https://github.com/inst-event-emitter/eventemitter-backend/pull/18/commits/f63f51034c6dc644adf3b8100c3aa5ca0cc31eae#diff-6b4ed25cf033c0ad808af3a83dd3f75cR19)

File: app/services/exportEventsService.js (hunk 3 above)
Comment (dkryvi) [reply app/services/exportEventsService.js]: [Done](https://github.com/inst-event-emitter/eventemitter-backend/pull/18/commits/f63f51034c6dc644adf3b8100c3aa5ca0cc31eae#diff-3dfd1aeed57246722026e05cc36b9a86R6)

File: app/config/dev.json (hunk 1 above)
Comment (dkryvi) [reply app/config/dev.json]: [Link](https://github.com/bee-queue/bee-queue#queuecheckstalledjobsinterval-cb) from docs. You can read and on Thursday we will discuss.

**Review by akosenko — APPROVED** [review]

**Pull request merged**